eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -46,13 +40,8 @@ jobs:
 
     strategy:
```

Comment:
We don't need the matrix here if we only have one thing in it, its just a unmatrixed job.